test/fixtures/doc_methods.yml

```diff
@@ -0,0 +1,21 @@
+migrate:
+  id: 1
+  repo_id: 2
+  name: undocumented_method
```

Comment:
Not sure if it's important to you to have real file and method names, but it makes the tests clearer to write it this way.